corehq/pillows/mappings/domain_mapping.py

```diff
@@ -1,7 +1,7 @@
 from corehq.util.elastic import prefix_for_tests
 from pillowtop.es_utils import ElasticsearchIndexInfo, DOMAIN_HQ_INDEX_NAME
 
-DOMAIN_INDEX = prefix_for_tests("hqdomains_2020-02-10")
+DOMAIN_INDEX = prefix_for_tests("hqdomains_2021-03-08")
```

Comment:
Just realized this. Did you mean to use 2021, or should this be `2022-03-08`?

Reply:
I did mean to use 2022, but it looks like I was consistently wrong throughout the PR, so it shouldn't matter.

Reply:
No need to change it, you've satisfied my curiosity :)